common/constants/default.go

```diff
@@ -25,6 +25,7 @@ const (
 const (
 	DefaultCAURL              = "/ca.crt"
 	DefaultCertURL            = "/edge.crt"
+	DefaultHasNodeNameURL     = "/node/{nodename}"
```

Comment:
Would it be better using `CheckNodeName` ?

Reply:
Yes, it was indeed called CheckNodeName before. However, later on we considered that this interface might not necessarily be used for node name checking in the future. Perhaps it can simply be used to determine whether a node exists or not.

Reply:
Or we can call it `CheckNode`? It'll be better expanded, not just for node name. But this name won't cause any impact, it's just my suggestion.

Reply:
Thank you for your suggestion. I have made corresponding modifications.